app/src/pages/CollectionsPage/CollectionsPage.tsx

```diff
@@ -0,0 +1,189 @@
+                  </Text>
+                </Stack>
+              </Box>
+              <Box variant='card'>
+                <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+                  <Box variant='card' width='250px' height='250px'>
+                    <Image source={collectionImageUrl} alternativeText='image' fitType='contain' />
+                  </Box>
+                  <Spacing variant={PaddingSize.Wide2} />
+                  <Text variant='header7'>
+                    {`${collectionDescription}sold for $37k yesterday`}
+                  </Text>
+                </Stack>
+              </Box>
+              <Box variant='card'>
+                <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+                  <Box variant='card' width='250px' height='250px'>
+                    <Image source={collectionImageUrl} alternativeText='image' fitType='contain' />
+                  </Box>
+                  <Spacing variant={PaddingSize.Wide2} />
+                  <Text variant='header7'>
+                    {`${collectionDescription}sold for $37k yesterday`}
+                  </Text>
+                </Stack>
+              </Box>
+            </EqualGrid>
+            <Stack direction={Direction.Horizontal} childAlignment={Alignment.Start} contentAlignment={Alignment.Center}>
+              <EqualGrid isFullHeight={false} childSizeResponsive={{ base: 10, small: 8, large: 6, extraLarge: 4 }} contentAlignment={Alignment.Center} childAlignment={Alignment.Center} shouldAddGutters={true}>
+                <Box variant='card' isFullWidth={true}>
+                  <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+                    <Text variant='header3'>10th percentile price</Text>
+                    <Text variant='header7'>Last 7 days</Text>
+                    <Text variant='header2'>187</Text>
+                  </Stack>
+                </Box>
+                <Box variant='card'>
+                  <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+                    <Text variant='header3'>Median price</Text>
+                    <Text variant='header7'>Last 7 days</Text>
+                    <Text variant='header2'>$2324</Text>
+                  </Stack>
+                </Box>
+                <Box variant='card'>
+                  <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+                    <Text variant='header3'>90th percentile price</Text>
+                    <Text variant='header7'>Last 7 days</Text>
```

Comment:
these should not be headers - only use headers for things that are actually titles. this is just basically a table you can just use `large` or `bold` or both (`bold-large`) variant instead